api-reference/v1.0/api/groupsettingtemplate_get.md

```diff
@@ -0,0 +1,72 @@
+# Get a group setting template
+
+A group setting template represents a template of settings from which settings may be created within a tenant. This operation allows retrieval of the properties of the [groupSettingTemplate](../resources/groupsettingtemplate.md) object, including the available settings and their defaults.
```

Comment:
I wonder whether we should also document the set of available settings - or at least in the example response, actually put the full real data response in.  That response is somewhat self documenting